packages/webpack/src/plugin.js

```diff
@@ -464,50 +464,72 @@ class LavaMoatPlugin {
                   )
                   return
                 }
-                diag.rawDebug(
-                  1,
-                  '> adding runtime (additionalChunkRuntimeRequirements)'
+                let runtimeChunks = []
+                if (
+                  chunk.name &&
+                  options.unlockedChunksUnsafe?.test(chunk.name)
+                ) {
+                  diag.rawDebug(
+                    1,
+                    `> adding UNLOCKED runtime for chunk ${chunk.name}`
+                  )
+                  runtimeChunks = [
+                    {
+                      name: 'ENUM',
+                      file: path.join(__dirname, './ENUM.json'),
```

Comment:
```suggestion
                      file: path.join(__dirname, 'ENUM.json'),
```

Reply:
suggestion:

- make string constants for filenames e.g., `const ENUM_FILENAME = 'ENUM.json'`
- other constants for other strings having significant names (e.g., `endowmentsToolkit`)

Reply:
When I refactor plugin.js into a few files, I intend to make this slightly more declarative maybe, but I don't see much value in more indirection in this as is.